main.cpp

```diff
@@ -220,15 +224,18 @@ int main()
     auto completedHomework = doHomework(100, 52.0, 100.00);
     
     //IGNORE FUNCTIONS
-    ignoreUnused(carRented);
-    ignoreUnused(workedForYears);
-    ignoreUnused(dogCaredFor);
-    ignoreUnused(finishedDinner);
-    ignoreUnused(eligibleForHiring);
-    ignoreUnused(madeInvestment);
-    ignoreUnused(songRecorded);
-    ignoreUnused(userSound);
-    ignoreUnused(completedHomework);
+    ignoreUnused
+    (
+        carRented,
+        workedForYears,
+        dogCaredFor,
+        finishedDinner,
+        eligibleForHiring,
+        madeInvestment,
+        songRecorded,
+        userSound,
+        completedHomework
+    );
```

Comment:
- [ ] revise.  don't use this style for passing multiple arguments to a function.